.gitignore

```diff
@@ -69,3 +69,6 @@
 
 # Unignore tests' bundle config
 !/Library/Homebrew/test/.bundle/config
+
+# Ignore text editor files
+*.swp
```

Comment:
That this hasn't been there tells me maybe the project expects this to be in the user's global gitignore…